cli/src/swam/cli/Main.scala

```diff
@@ -340,25 +340,38 @@ object Main extends CommandIOApp(name = "swam-cli", header = "Swam from the comm
                           val functionName =
                           compiled.names.flatMap(_.subsections.collectFirstSome {
                             case FunctionNames(names) =>
+                              
+                              val mapList = names.values
+                              val check = mapList.exists(j => {j == func_name})
+                              if(!check){
+                                System.err.println("Function name does not exists.")
+                                sys.exit(1)
```

Comment:
Take a look at how this is handled in a CommandIOApp. You never should directly exit in this way

Reply:
Checked with ExitCode.Error but it not stopping the execution at that point. But checked you refactored code and understood that the exit code needs to return so that CommandIOApp can capture it and tell the status whether the status of execution is Success or Error